deploy.yml

```
@@ -13,12 +13,12 @@
       debug: msg="==== Starting deploy playbook ===="
       tags: deploy
 
+    - import_tasks: tasks/set_vars.yml
+
     - name: Check if apps_filter is defined
       fail:
         msg: "Deploy playbook must be used with apps_filter defined"
```

Maybe change the message to reflect this new rule?
```suggestion
         msg: "This project contains several apps. Please list which apps you inted to deploy by defining `apps_filter` option."
```